### PR TITLE
:bookmark: Release 2.1.0

### LINKED ISSUE #198: Release a new patch/minor version of Open Klant
### Thema / Theme

Other

### Omschrijving / Description

With the last version of Open Klant from early March, I think it's time for an update of OK v2. 

### Toegevoegde waarde / Added value

_No response_

### Aanvullende opmerkingen / Additional context

_No response_

### PR DESCRIPTION
**Pull request by Coperh**
Fixes #198

Anything missing from the changelog?

### CODE REVIEW

File: .bumpversion.cfg [hunk 1]
```diff
@@ -1,20 +1,12 @@
...
 search = "version": "{current_version}",
 replace = "version": "{new_version}",
 
 [bumpversion:file:src/openklant/__init__.py]
-
-# [bumpversion:file:docs/api/index.rst]
-
-# [bumpversion:file:publiccode.yaml]
-# search = softwareVersion: {current_version}
-# replace = softwareVersion: {new_version}
```
Comment (Coperh) [inline .bumpversion.cfg:20]: These were removed automatically. IDK if that matters

**Comment by Coperh** [comment]
I also do want #199 merged before release.

File: .bumpversion.cfg (hunk 1 above)
Comment (annashamray) [reply .bumpversion.cfg:20]: Sure let's remove commented code

**Review by annashamray — APPROVED** [review]

**Comment by Coperh** [comment]
@stevenbal @bart-maykin do you guys see anything missing from the change list or bad change descriptions or is everything A-OK?

**Review by stevenbal — CHANGES_REQUESTED** [review]
I think https://github.com/maykinmedia/open-klant/issues/183 is missing from the changelog

**Comment by Coperh** [comment]
Does open-klant not have documentation?

**Comment by annashamray** [comment]
@Coperh You mean RTD?

**Comment by Coperh** [comment]
@annashamray Yes

**Comment by annashamray** [comment]
Good question! @bart-maykin Do you know something?

**Review by stevenbal — APPROVED** [review]

**Comment by stevenbal** [comment]
@Coperh there is a RTD apparently: https://open-klant.readthedocs.io/en/latest/, not much in it yet though

**Comment by joeribekker** [comment]
@Coperh sorry to join the party late, but I see that 2 people approved this PR but I see something in the README that caught my eye:

The 2.0.0 was changed to 2.1.0. This is not correct imo.

There should be an entry with 2.0.0, showing it offered API version `0.0.3`. There should be another entry showing 2.1.0 offers `0.0.3+` since it contains an experimental field. This field in the changelog is shown as `afgehandeld_op`, but should show `afgehandeldOp` which is how the API spec works.

**Pull request merged**